atlasdb-client/src/main/java/com/palantir/atlasdb/keyvalue/impl/ProfilingKeyValueService.java

```diff
@@ -380,6 +382,17 @@ public void checkAndSet(CheckAndSetRequest request) {
                 logCellsAndSize("checkAndSet", request.table(), 1, request.newValue().length));
     }
 
+    @Override
+    public void multiCheckAndSet(MultiCheckAndSetRequest multiCheckAndSetRequest) throws MultiCheckAndSetException {
+        maybeLog(
+                () -> delegate.multiCheckAndSet(multiCheckAndSetRequest),
+                logCellsAndSize(
+                        "multiCheckAndSet",
+                        multiCheckAndSetRequest.tableRef(),
+                        1,
+                        multiCheckAndSetRequest.updates().size()));
```

Comment:
Do we want the number of updates or the total sum of sizes? Seems to me like we want to have the former be the third parameter, and the total size the fourth